pages/newsletter.js

```diff
@@ -54,13 +54,13 @@ export default function Newsletter() {
                     <div className="mt-4 font-semibold text-primary-medium dark:text-primary-low">
                       Regular information
                     </div>
-                    <div className="mt-2 leading-7 text-gray-400">
+                    <div className="mt-2 leading-7 text-primary-medium-low">
                       This will include latest features, upcoming releases and
                       offers.
                     </div>
                   </div>
                   <div className="flex flex-col items-start">
-                    <div className="rounded-md bg-white/5 p-2 ring-1 ring-white/10">
+                    <div className="rounded-md bg-primary-medium p-2 ring-1 ring-primary-medium">
```

Comment:
I think the icons are now not visible because the background, ring and icon colour are all the same, please see screenshot...

<img width="948" alt="Screenshot 2023-06-25 at 09 42 49" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/a7b7c539-d635-4ff2-992f-205bbec44c68">